src/components/Modal/style.scss

```diff
@@ -70,3 +70,15 @@
 .content {
   @extend %layout--bounds;
 }
+
+.pageActions {
+  display: flex;
+
+  * {
```

Comment:
You sure this is not too broad? Performance and future additions (of inside content) might suffer.

Reply:
I was wondering about this too. I'm wondering if we'll ever have other content in the actions.

Reply:
@nickstamas ^